### PR TITLE
Cover systemd testing with testsuite

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
## Description

- Added only for Tumbleweed for now

## Ticket

- [poo#23798](https://progress.opensuse.org/issues/23798)

## Based on

- See discussion here: [gh#os-autoinst-distri-opensuse/3364](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3364)

### CODE REVIEW

**Comment by okurz** [comment]
Please start commit message subject lines with capital letter

File: tests/console/systemd_testsuite.pm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Summary: Run testsuite included in systemd sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
```
Comment (okurz) [inline tests/console/systemd_testsuite.pm]: skip the `()`

**Comment by SergioAtSUSE** [comment]
Verification run: http://copland.arch.suse.de/tests/1540#step/systemd_testsuite/26

The failure is on the systemd-testsute side, not this test module itself. (CC @tblume)

**Comment by SergioAtSUSE** [comment]
Here is "passed" when checking that 17 tests are failing: http://copland.arch.suse.de/tests/1537

**Comment by SergioAtSUSE** [comment]
@tblume told me that I was installing an old version. The package was renamed to systemd-v234-testsuite

**Comment by SergioAtSUSE** [comment]
New verification run: http://copland.arch.suse.de/tests/1544

**Pull request merged**